include/nbl/video/IDriverMemoryAllocation.h

```diff
@@ -90,12 +90,13 @@ class IDriverMemoryAllocation : public virtual core::IReferenceCounted
         };
 
         //! Memory allocate flags
-        enum E_MEMORY_ALLOCATE_FLAGS
+        enum E_MEMORY_ALLOCATE_FLAGS : uint32_t
         {
             EMAF_NONE = 0x00000000,
             EMAF_DEVICE_MASK_BIT = 0x00000001,
             EMAF_DEVICE_ADDRESS_BIT = 0x00000002,
             EMAF_DEVICE_ADDRESS_CAPTURE_REPLAY_BIT = 0x00000004,
+            // Mapping Capability Flags
```

Comment:
huh?

Reply:
Don't worry about this, I was going to move the mapping capability flags here, but saw that it was not trivial so I decided to do other tasks first